Makefile

```diff
@@ -21,37 +21,23 @@ SHELL := /bin/bash
 # Current version, updated after a release.
 VERSION ?= "0.5.0"
 
-# If GOPATH is not set by the env, set it to a sane value
-GOPATH ?= $(shell cd ../../..; pwd)
-export GOPATH
-
-# If GOPATH is made up of several paths, use the first one for our targets in this Makefile
-GO_TOP := $(shell echo ${GOPATH} | cut -d ':' -f1)
-
-export CGO_ENABLE=0
-
 # OUT is the directory where dist artifacts and temp files will be created.
-OUT=${GO_TOP}/out
+OUT := out
```

Comment:
This affects non-mac platforms - please don't change the way we build for other platforms, and I don't think mac needs to build in the source tree either.